cython_setuptools/vendor.py

```diff
@@ -1,8 +1,9 @@
-import subprocess
+import argparse
+import configparser
```

Comment:
`configparser` import is introduced in this commit ?

Reply:
Resolved, Thanks